app/config/bootstrap/cache.php

```diff
@@ -83,15 +83,15 @@
 		if (!(($connection = Connections::get($name)) instanceof Database)) {
 			continue;
 		}
-		Filters::apply($connection, 'describe', function($params, $chain) use ($name) {
```

Comment:
This is done in a separate commit https://github.com/UnionOfRAD/framework/pull/41/commits/cdbc494a9da4fb2c18c4651b8de4fa1e5ec7e05d